index.js

```diff
@@ -1,22 +1,349 @@
-const openBtn = document.querySelector('#open-menu-btn');
-const closeBtn = document.querySelector('#close-menu-btn');
-const menu = document.querySelector('.mobile-menu');
-const listBtns = document.querySelectorAll('.listBtns');
+const openBtn = document.querySelector("#open-menu-btn");
+const closeBtn = document.querySelector("#close-menu-btn");
+const menu = document.querySelector(".mobile-menu");
+const listBtns = document.querySelectorAll(".listBtns");
+const body = document.querySelector("body");
+const seeMoreBtn = document.querySelector("#seeMoreBtn");
 
-function createClass (elem, className) {
-    elem.classList.add(className)
+function creatPopUp(
+  name,
+  description,
+  img,
+  techs,
+  live = "SeeLive",
+  source = "SeeSource"
+) {
+  const wrapper = document.createElement("div");
+  const contentContainer = document.createElement("div");
+  const headerContainer = document.createElement("div");
+  const header = document.createElement("h2");
+  header.textContent = name;
+  const closeBtn = document.createElement("a");
+  closeBtn.textContent = "x";
+  const image = document.createElement("img");
+  image.src = img;
+  const paragraph = document.createElement("p");
+  paragraph.textContent = description;
+  const tags = document.createElement("div");
+  const anchor1 = document.createElement("a");
+  const anchor2 = document.createElement("a");
+  const anchor3 = document.createElement("a");
+  anchor1.textContent = techs[0];
+  anchor2.textContent = techs[1];
+  anchor3.textContent = techs[2];
+  anchor1.href = "#";
+  anchor2.href = "#";
+  anchor3.href = "#";
+  const btnLive = document.createElement("button");
+  const btnSrc = document.createElement("button");
+  btnLive.textContent = live;
+  btnSrc.textContent = source;
+
+  wrapper.setAttribute(
+    "style",
+    "position:fixed; top:0; bottom: 0; left:0; right:0; display: flex; justify-content: center; align-items: center; background: rgba(0, 0, 0, 0.8); backdrop-filter: blur(8px);"
+  );
```

Comment:
- Following the original design is one of the project's requirements as it's an important skill for a font-end dev. The pop up looks a little bit different from the original design: 
Your webpage : 

![image](https://user-images.githubusercontent.com/43698511/136275069-e0988142-86f3-4512-8a59-aba3d971c461.png)

Original design:
![image](https://user-images.githubusercontent.com/43698511/136275117-6807ba4a-651c-4c72-b764-dffb030cad2c.png)

- [ ] As you can see from the pictures, the text is centered the buttons are inlined. Kindly make the required changes to make it look like the original design.
- [ ] The project's image also looks different from the original design. The one on your webpage looks bigger I think it needs resizing. Kindly fix the issue.

- [ ] [OPTIONAL] I think it's better to put all the styling in a CSS file. You can use CSS classes and then add them to the element using `wrapper.classList.add("myclass-name");`